tests/Integration/Behaviour/Features/Context/Domain/AbstractDomainFeatureContext.php

```diff
@@ -77,6 +77,10 @@ protected function getContainer(): ContainerInterface
      */
     protected function referencesToIds(string $references): array
     {
+        if (empty($references)) {
```

Comment:
Hmm again, I thought I already saw you adding this modification 🤔

Reply:
yeah it was in some other PR (I don't remember which, but its probably still not merged and it keeps bothering me.. what can I do :man_shrugging: )